app/styles/app.scss

```diff
@@ -50,18 +51,21 @@ p
 }
 .navbar
 {
-  background-color: transparent;
-  background: transparent;
   border-color: transparent;
-  position: absolute;
+  position: fixed;
```

Comment:
I would say this should be it's own class so it doesn't mess with the navbar that we will be getting from ember-osf eventually.